src/main/java/cms/gov/madie/measure/resources/MeasureController.java

```diff
@@ -177,4 +176,12 @@ public ResponseEntity<String> getMeasureBundle(
   private boolean isCqlLibraryNameChanged(Measure measure, Optional<Measure> persistedMeasure) {
     return !Objects.equals(persistedMeasure.get().getCqlLibraryName(), measure.getCqlLibraryName());
   }
+
+  private boolean isCqlLibraryMeasurementPeriodChanged(
```

Comment:
This is related to the measurement period. I think we can remove cqlLibrary things from method name  `isMeasurementPeriodChanged`